docs/Integration-structure.md

```diff
@@ -159,7 +159,7 @@ nginX
 
 ```
   "version": {
-        "integ": "0.1.0",
+        "integration": "0.1.0",
```

Comment:
I wonder if there's a way we can better clarify that it's the current integration (or do we mean the integration plugin)? Would it be wise or viable to set the key to the integration name?

Reply:
not sure that renaming it will make a difference